sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala

```diff
@@ -304,4 +304,14 @@ class UDFSuite extends QueryTest with SharedSQLContext {
     assert(explainStr(spark.range(1).select(udf1(udf2(functions.lit(1)))))
       .contains(s"UDF:$udf1Name(UDF:$udf2Name(1))"))
   }
+
+  test("SPARK-23666 Do not display exprId in argument names") {
+    Seq(((1, 2), 3)).toDF("a", "b").createOrReplaceTempView("x")
```

Comment:
wrap with `withTempView`

Reply:
I also wrapped other tests in this file cuz they didn't drop temp views.